sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2310,10 +2310,12 @@ pi_result hip_piextQueueGetNativeHandle(pi_queue queue,
 /// \return TBD
 pi_result hip_piextQueueCreateWithNativeHandle(pi_native_handle nativeHandle,
                                                pi_context context,
-                                               pi_queue *queue,
-                                               bool ownNativeHandle) {
+                                               pi_device *device,
```

Comment:
Why is it a pointer to the `pi_device` rather than a value? 
NOTE: the `pi_device` is itself a pointer.

Reply:
Thank you for reminding me. You are right, there is really no point in having it be a pointer to `pi_device` and it has been changed to be a `pi_device` argument instead.